db/migrate/20240303144633_add_contact_info_to_donation_sites.rb

```diff
@@ -0,0 +1,7 @@
+class AddContactInfoToDonationSites < ActiveRecord::Migration[7.0]
+  def change
+    add_column :donation_sites, :contact_name, :string
```

Comment:
Likely this is fine, but best practices in migrations is to minimize the amount of statements that will lock the table. 

The alternative is to do something like 
```
change_table :dontation_sites do |t|
  t.string :contact_name
  t.string :email,
  t.string :phone
end
```